tests/test_elasticmock.py

```diff
@@ -206,3 +206,7 @@ def test_should_return_suggestions(self):
                 }
             ],
         }, suggestion)
+
+
+if __name__ == '__main__':
```

Comment:
You don't need to do this to run the tests. You can run `make test` from the root folder :)